client/components/EventForm/index.jsx

```diff
@@ -441,37 +444,26 @@ const mapStateToProps = (state) => ({
     unlockPrivilege: selectors.getPrivileges(state).planning_unlock ? true : false,
 })
 
+const dispatchSubmit = (dispatch, event) => dispatch(actions.askConfirmationBeforeSavingEvent(event))
```

Comment:
This can be an independent action in the actions folder rather than a const function here.